### DISCUSSION TITLE
Fallback for Emote's Translation keys

### DISCUSSION THREAD

**Discussion by Amirhan-Taipovjan-Greatest-I** [KosmX/emotes#407] [Ideas]
Hello!
Minecraft 1.19.4 (and newer by date) has the fallback feature that allows to use message (placeholder) from "fallback" if translation keys are not working.

With
```
    "name": {"translate":"emotecraft.nameTranslation.Test", Fallback:"Testie"},
    "description": {"translate":"emotecraft.descriptionTranslation.Test", Fallback:"Testies are from Testie =)"},
```
In .json file should appear in-game as "Testie" (not as "emotecraft.descriptionTranslation.Test", because it is a translation key).

It can be useful if someone want to make a localization for their Emote Packs.

**Comment by dima-dencep** [comment]
https://minecraft.wiki/w/Text_component_format
